frontend/src/components/layout/CircleBackground.tsx

```diff
@@ -3,11 +3,13 @@ import styled from 'styled-components';
 import { ThemeConfig } from '../config/Theme';
 import { FloatingCircle, Coordinate } from '../special/FloatingCircle';
 import Header from '../navigation/Header';
-import { MainContainer, LandingPageContainer } from '../core/Container';
+import { MainContainer, CircleBackgroundContainer } from '../core/Container';
 
 const Content = styled.div`
+  position: relative;
   width: 100%;
-  min-height: 100vh;
+  height: 100vh;
```

Comment:
I remember that I ran into issues on my PR with using vw - is there any need to change this or no? Not a requested change lol, just genuinely asking

Reply:
The idea behind this was to have the circle positioning depend on the entire view size of the screen - however ik you brought up the circles in the screencast so we can also talk about it there (whether it should be changed to static to keep constant regardless of screen size for example)